### PR TITLE
refactor(logging): debrand stackdriver logging in favor of cloud logging

### PR DESCRIPTION
**Pull request by nicoleczhu**
Requested by product team, Stackdriver Logging was renamed to Cloud Logging and we shouldn't use "Stackdriver" as a brand name anymore. 

Update: Created a [CL](https://critique-ng.corp.google.com/cl/337147495) to update accompanying documentation to these samples

### CODE REVIEW

File: getting-started/bookshelf/bookshelf.go [hunk 1]
```diff
@@ -62,8 +62,8 @@ type Bookshelf struct {
 	// logWriter is used for request logging and can be overridden for tests.
 	//
 	// See https://cloud.google.com/logging/docs/setup/go for how to use the
-	// Stackdriver logging client. Output to stdout and stderr is automaticaly
-	// sent to Stackdriver when running on App Engine.
+	// Cloud logging client. Output to stdout and stderr is automaticaly
```
Comment (tbpg) [inline getting-started/bookshelf/bookshelf.go:65]: Capitalize "logging"?

**Review by tbpg — COMMENTED** [review]
Thanks! One question, otherwise looks good!

File: getting-started/bookshelf/bookshelf.go (hunk 1 above)
Comment (nicoleczhu) [reply getting-started/bookshelf/bookshelf.go]: Good catch. It is capitalized

**Review by tbpg — APPROVED** [review]
Thanks!

**Pull request merged**